quint/io-cli-tests.md

```diff
@@ -686,6 +689,35 @@ exit $exit_code
 ]
 ```
 
+### Test does not mean assignments (#1133)
+
+See: https://github.com/informalsystems/quint/issues/1133
+
+FIXME
```

Comment:
Could you add a note indicating what is meant to be fixed?

Reply:
Seeing how many FIXMEs are in this test: is the test testing anything useful currently? Or is just a sekeleton of a test that should be failing as it, but will be expanded to test something in the future? If the latter, I suggest leaving this test out of the PR.

Reply:
It's a test I have added when I started working on #1133. Since it is the next issue on my list, I would prefer keeping it in the imperfect shape rather than stashing it somewhere on my disk.